### PR TITLE
[SUGGESTION] Ask for testing instructions in PRs

### PR DESCRIPTION
**Pull request by pafonta**
## Description

It would be nice that all PRs contain instructions on how to test that a PR does what it says.

It would help reviewers and therefore would make reviews faster.

It would also help the assignee to check that all the said changes were done.

Here is the suggested new PR template:

suggested PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
+Fixes #{issue-id-number}.
+
 ## Description
 
 Please provide here a summary of the changes introduced by this PR.
 
-Fixes #{issue-id-number}.
+## How to test?
 
+Please provide here instrutions on how to test the changes introduced by this PR.
```
Comment (FrancescoCasalegno) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
Please provide here instructions on how to test the changes introduced by this PR.
```
Small typo.

**Review by FrancescoCasalegno — APPROVED** [review]
LGTM, only a small typo!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (pafonta) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nice catch! Thanks!

**Review by EmilieDel — APPROVED** [review]

**Review by Stannislav — APPROVED** [review]
Good suggestion, thanks

**Pull request merged**